extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -385,6 +387,56 @@ describes.realWin(
       );
     });
 
+    it('should send attribution source ping when attribution reporting is enabled', async () => {
```

Comment:
should we add a test that shows the param is not replaced if the browser does not support ?